app/views/alaveteli_pro/info_requests/_embargo_form.html.erb

```diff
@@ -8,6 +8,8 @@
              locals: {embargo: info_request.embargo, tense: :present} %>
 
   <label class="houdini-label" for="input1"><%= _("Change privacy") %></label>
+  <span style="font-size:13px">or</span>
+  <label class="houdini-label is--bold" for="input2"><%= _("Publish now") %></label>
```

Comment:
How about just doing?

```diff
- <label class="houdini-label" for="input1"><%= _("Change privacy") %></label>
+ <label class="houdini-label" for="input1"><%= _("Change privacy or <strong>publish now</strong>") %></label>
```

Reply:
We just decided on standup to keep these actions as two individual links that should unfold the correct options (i.e. clicking "publish now" only unfolds the "publish now" button). We wanted to keep the two-step process so that it's clear the user is taking an intentional step to publish as it's not easy to revert.